src/jemalloc/include/jemalloc/internal/valgrind.h

```diff
@@ -38,9 +38,10 @@
     ptr_maybe_null, old_ptr, old_usize, old_rzsize, old_ptr_maybe_null,	\
     zero) do {								\
 	if (in_valgrind) {						\
+		size_t rzsize = (ptr != NULL) ? p2rz(ptr) : 0;		\
```

Comment:
I'm not sure I understand the need for this change. The `ptr` variable can't be NULL where the calls happened.